public/gameInfo.php

```diff
@@ -1146,16 +1146,16 @@ function ResetProgress() {
                         }
 
                         if ($claimExpiration) {
-                            $claimFormattedDate = $claimExpiration->format('d M Y, H:i');
                             $isAlreadyExpired = Carbon::parse($claimExpiration)->isPast();
+                            $isAlreadyExpiredCopy = $isAlreadyExpired ? "Expired" : "Expires";
```

Comment:
I'd just merge the two lines. `$isAlreadyExpiredCopy` is confusing.
```
$isAlreadyExpired = Carbon::parse($claimExpiration)->isPast() ? "Expired" : "Expires";
```

Reply:
Good idea, resolved in https://github.com/RetroAchievements/RAWeb/pull/1473/commits/4c9b5f8c299a109339a54e213fed620bc2a319b7